js/map.mjs

```diff
@@ -228,7 +228,7 @@ function initOverlay(){
     });
 
     runOnTree(jsondata.nirnroot, function(nirn){
-        if(nirn.cell == "Outdoors"){
+        if(nirn.name){
```

Comment:
ideally we'd split the "nirnroot on map" from "has a name" so we can put unnamed nirns on the map, but for now this is fine. We'll deal with the problem of unnamed nirns when we come to it.